packages/cosmwasm-stargate/types/codec/gogoproto/gogo.d.ts

```diff
@@ -0,0 +1 @@
+export declare const protobufPackage = "gogoproto";
```

Comment:
Do we need this package?

Reply:
No, and it seems there are a bunch of other packages we don't need. 🤔